### PR TITLE
gitea: update to 1.16.5

### PR DESCRIPTION
**Pull request by zen0bit**
- I tested the changes in this PR: **briefly**

- I built this PR locally for my native architecture, (x64 libc)

### CODE REVIEW

File: srcpkgs/gitea/template [hunk 1]
```diff
@@ -32,7 +32,7 @@ license="MIT"
 homepage="https://gitea.io"
 changelog="https://github.com/go-gitea/gitea/blob/master/CHANGELOG.md"
```
Comment (paper42) [inline srcpkgs/gitea/template:33]: This should link to a raw file.

File: srcpkgs/gitea/template [hunk 2]
```diff
@@ -32,7 +32,7 @@ license="MIT"
...
 distfiles="https://github.com/go-gitea/gitea/releases/download/v${version}/gitea-src-${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/gitea/template:34]: Could we just use https://dl.gitea.io/gitea/?

File: srcpkgs/gitea/template (hunk 1 above)
Comment (zen0bit) [reply srcpkgs/gitea/template]: https://raw.githubusercontent.com/go-gitea/gitea/main/CHANGELOG.md

File: srcpkgs/gitea/template (hunk 2 above)
Comment (zen0bit) [reply srcpkgs/gitea/template]: changed...
good?

**Pull request merged**